src/frontend/src/lib/components/ui/TextWithDescriptionAndLogo.svelte

```diff
@@ -0,0 +1,19 @@
+<script lang="ts">
+	import { replacePlaceholders } from '$lib/utils/i18n.utils';
+	import { i18n } from '$lib/stores/i18n.store';
+	import Logo from '$lib/components/ui/Logo.svelte';
+
+	export let name: string;
+	export let description: string | undefined;
```

Comment:
You can assign a default value `undefined` that way you are not forced to use the property in consumers.

```
// before

<NetworkButton {id} {name} {icon} on:icSelected description={undefined} />

// after


<NetworkButton {id} {name} {icon} on:icSelected />
```

Reply:
Done 👍